db/migrate/20160228205806_create_services.rb

```diff
@@ -0,0 +1,13 @@
+class CreateServices < ActiveRecord::Migration
+  def change
+    create_table :services do |t|
+      t.integer :student_id
+      t.integer :educator_id
+      t.integer :service_type_id
+      t.text :text
+      t.datetime :recorded_at
```

Comment:
This might be a useful thing to add, but we also need the date that the educator started providing the service as well.  In the UI, it's the "When did they start?" date:
<img width="592" alt="screen shot 2016-02-29 at 12 32 25 pm" src="https://cloud.githubusercontent.com/assets/1056957/13402921/87eed5d0-dee0-11e5-99cb-c98f25436251.png">

This is different than the date the user recorded this information, so we can add a `date_started` field here to capture this.